Formula/bnl.rb

```diff
@@ -1,13 +1,11 @@
 class Bnl < Formula
   desc "Blendle Command Line Tools"
   homepage "https://github.com/blendle/bnl"
-  url "https://github.com/blendle/bnl.git",
+  url "git://github.com/blendle/bnl.git",
```

Comment:
`git://` is an insecure protocol. I think you could get `https://` to work by adding your username and pass to `~/.netrc`, otherwise see if you could force this URL to use SSH (maybe `git+ssh://`?).